config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -253,12 +253,6 @@
   <!-- until https://github.com/checkstyle/checkstyle/issues/13839 -->
   <suppress id="propertiesMacroMustExist"
          files="src[\\/]xdocs[\\/]checks[\\/]annotation[\\/]suppresswarningsholder.xml.template"/>
-  <!-- until https://github.com/checkstyle/checkstyle/issues/13893 -->
-  <suppress id="propertiesMacroMustExist"
-         files="src[\\/]xdocs[\\/]checks[\\/]coding[\\/]illegaltype.xml.template"/>
-  <!-- until https://github.com/checkstyle/checkstyle/issues/13930 -->
```

Comment:
Why are we working on 2 different issues in this one commit/PR?

Reply:
> Why are we working on 2 different issues in this one commit/PR?

About this, this issue has already been close. I found out it to be not removed from the suppression. Hence, removed it. I can revert it back to original.

Reply:
> this issue has already been close

I am ok that this removal remains. Please ping @romani in the issue that is already closed that the issue is done too.

Reply:
https://github.com/checkstyle/checkstyle/pull/13993#issuecomment-1798832847. i have done that. @romani  can confirm if the issue is really closed or not.

Reply:
I am ok to remove this leftover here

Reply:
this update was merged in separate commit